lib/src/box/types.ts

```diff
@@ -31,8 +31,10 @@ type Props = {
    */
   margin?: Space | Margin;
   /**
-   * Size of the padding to be applied to the custom area ('xxsmall' | 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge' | 'xxlarge').
+   * @deprecated This prop will be removed shortly, consider using the Inset component for this purpose.
+   *  Size of the padding to be applied to the custom area ('xxsmall' | 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge' | 'xxlarge').

```

Comment:
Remove the empty space at the begging of the sentence " Size of the padding...".